bigquery_etl/metadata/parse_metadata.py

```diff
@@ -59,7 +59,8 @@ class PartitionMetadata:
 
     field: str
     type: PartitionType
-    require_partition_filter: bool = attr.ib(False)
+    require_partition_filter: bool = attr.ib(True)
```

Comment:
The issue mentions a preference to set `require_partition_filter` to true by default. Until now, false was the default.

Reply:
I made the changes to explicitly list the configuration fields in the metadata files. I think it makes sense to set it to `true` by default. I guess the default values could be seen as recommendations and we usually recommend requiring partition filters.